core/src/tracing.rs

```diff
@@ -99,7 +99,7 @@ pub mod server {
 }
 
 /// Find the next char boundary to truncate at.
-fn truncate_at_char_boundary(s: &str, max: usize) -> &str {
+pub fn truncate_at_char_boundary(s: &str, max: usize) -> &str {
```

Comment:
Is this useful outside the crate now then? Is it exposed to help people writing custom loggers?

Reply:
I moved the this the rpc logger stuff so not needed to be public